### ISSUE TITLE
fichier non commité après nouvelle installation

### ISSUE THREAD

**Issue by RomainBaghi** [PnX-SI/GeoNature#725]
Bonjour, 

Juste pour info, si je fait un `git status` après une installation neuve de geonature, j'ai ce fichier qui attend d'être commité : `frontend/npm-shrinkwrap.json`
à rajouter dans le `.gitignore` ?

merci

**Comment by TheoLechemia** [comment]
Non, ce fichier ne doit pas être gitignoré. Il permet qu'on ai tous exactement le même arbre de dépendances JS. 
Je ne sais pas pourquoi il a été modifié chez toi ? Peut-être un npm install qui a été lancé après l’installation.

**Comment by RomainBaghi** [comment]
ok, non je ne sais pas non plus pourquoi il a été modifié.

**Comment by jbrieuclp** [comment]
Une possible explication : les versions des librairies indiquées dans le package.json ne sont pas fixées, ou seulement sur leur version majeure, du coup s'il y a des maj mineures pour un package, elles seront téléchargées et installées lors du npm install, le fichier npm-shrinktruc sera modifié en conséquence des nouvelles versions des libs.
De mon côté ce fichier me lourde lorsque je switche entre angular 4 et angular 7, mais jusqu'à présent un git checkout dessus pour annuler ses (auto)modifications ne m'ont pas posé problème. Il ne faut juste pas le supprimer, le fichier package-lock.json viendra le remplacer au prochaine npm install.
